docs/source/augly.image.rst

```diff
@@ -0,0 +1,61 @@
+augly.image package
+===================
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   augly.image.utils
```

Comment:
since augly.image.utils doesn't exist anymore as an .rst file, should this be deleted too?

Reply:
@jbitton Yep I'll delete that